jsapp/js/account/stripe.api.ts

```diff
@@ -10,14 +14,29 @@ export interface BaseProduct {
   metadata: {[key: string]: string};
 }
 
+export enum RecurringInterval {
+  Year = 'year',
+  Month = 'month',
+}
+
+enum UsageType {
+  Metered = 'metered',
+  Licensed = 'licensed',
+}
```

Comment:
That's fair! I went with enums instead of a type because they're accessible at run time - so it's a little easier to, say, verify that info coming from Stripe matches what we expect it to. But that's speculative at this point, so I'll refactor it.